src/PrestaShopBundle/Tests/resources/translations/en-US/AdminActions.en-US.xlf

```diff
@@ -0,0 +1,354 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I will try to use lighter xlf file, just wanted to be sure my stupid unit test retrieve the keys :)